ansible/playbook/tasks/update.yml

```diff
@@ -160,3 +160,11 @@
 
     - name: Check supervisord log
       debug: var=maintail_result.stdout_lines
+
+    # New config will be applied after next reboot.
+    - name: Configure supervisor       | Update supervisord.service
+      when: ansible_virtualization_type != "docker"
+      copy:
+        src: supervisord.service
+        dest: /usr/lib/systemd/system/
```

Comment:
Custom(ized)/overriden systemd unit files should be placed into `/etc/systemd/system`, so they won't be overwritten by package update.

Reply:
There is no package that creates that file – we do it with Ansible in `percona-images`. That means that there are two `supervisord.service` files now. Do we really need both?